lua/weapons/e.r.c.s.lua

```diff
@@ -0,0 +1,97 @@
+SWEP.Author = "el_patito_loco"
+SWEP.PrintName = "E.R.C.S" --"Experimental Rotating Chamber Sniper"
+SWEP.Category = TASWeapons.Category
+SWEP.Author = TASWeapons.Authors.El_Patito
+SWEP.Instructions = [[Left Click shoots a powerfull 12.7 mm sniper round that will destroy your enemies!!
+At the moments our engineers are working on new features wich will provide more mass destruction!!
+]]
+
+SWEP.Spawnable = true
+SWEP.AdminOnly = false
+
+SWEP.Primary.ClipSize = 10
+SWEP.Primary.DefaultClip = 10
+SWEP.Primary.Automatic = true
+SWEP.Primary.Ammo = "357"
+
+SWEP.Secondary.ClipSize = -1
+SWEP.Secondary.DefaultClip = -1
+SWEP.Secondary.Automatic = true
+SWEP.Secondary.Ammo = "none"
+
+SWEP.Weight = 5
+SWEP.AutoSwitchTo = false
+SWEP.AutoSwitchFrom = false
+
+SWEP.Slot = 4
+SWEP.SlotPos = 3
+SWEP.DrawAmmo = false
+SWEP.DrawCrosshair = true
+
+SWEP.ViewModel = "models/weapons/v_357.mdl"
+SWEP.WorldModel = "	models/weapons/w_357.mdl"
+
+local ShootSound = Sound("weapons/mortar/mortar_fire1.wav")
+
+SWEP.LoadSound = "weapons/ammopickup.wav"
+
+SWEP.Primary.Recoil = 1
+SWEP.Primary.Damage = 75
+SWEP.Primary.NumShots = 1
+SWEP.Primary.Spread = 0
+SWEP.Primary.Cone = 0.25
+SWEP.Primary.Delay = 1
+SWEP.Tracer = 4
+SWEP.Penetration = true
+SWEP.Ricochet = true
+SWEP.MaxRicochet = 100
+--SWEP.Scoped = false
+SWEP.MouseWasDown = true
+
+SWEP.RapidFire = false
+SWEP.RapidFireTimer = 0
+SWEP.RapidFireForce = 1000000
+
+function SWEP:Initialize()
+	self:SetHoldType("revolver")
+end
+
+function SWEP:PrimaryAttack()
+	local ply = self:GetOwner()
+
+	local Bullet = {}
+	Bullet.Num = self.Primary.NumShots
+	Bullet.Src = ply:GetShootPos()
+	Bullet.Dir = ply:GetAimVector()
+	Bullet.Spread = Vector(self.Primary.Spread, self.Primary.Spread, 0)
+	Bullet.Tracer = self.Primary.Tracer
+	Bullet.Damage = self.Primary.Damage
+	Bullet.AmmoType = self.Primary.Ammo
+	Bullet.Ricochet = self.Primary.Ricochet
+	Bullet.MaxRicochet = self.Primary.MaxRicochet
+	Bullet.Penetration = self.Primary.Penetration
+
+	local rnda = self.Primary.Recoil * -1
+	local rndb = self.Primary.Recoil * math.random(1, 0)
```

Comment:
The parameters of `math.random(a, b)` are a range, so should be in ascending order. `math.random(1, 0)` always return `1` and never `0`.

Reply:
Also solved now its math.random(0,1)